libraries/AP_HAL_ChibiOS/hwdef/TBS_LUCID_H7_WING/README.md

```diff
@@ -0,0 +1,134 @@
+# TBS Lucid H7 Wing Flight Controller
+
+The TBS Lucid H7 Wing is a flight controller produced by [TBS](https://www.team-blacksheep.com/).
+
+## Features
+
+ - MCU: STM32H743VIH6, 480MHz, 2MB Flash
+ - Input voltage: 8V-50.4V (3-12S)
+ - Output power: 8A cont./10A peak BEC. 5V, 6V, 8.4V (selectable)
+ - Channels: 13x PWM (including LED)
+ - Gyro: ICM42688 x 2 (SPI1 & SPI4)
+ - Baro: DPS310 (I2C2)
+ - Peripherals: 7x UART, 2x I2C, 1x CAN
+ - Measurements: 6x ADC (Vbat, Current, CB2, CU2, RSSI, AirSpeed)
+ - High speed ports: MicroSD BlackBox, SPI3 on pin headers
+ - Analog FPV: Switchable dual camera inputs, AT7456E OSD, 5V/9V/12V support at 5A (selectable)
+ - Digital FPV: "DJI"-connector with SBUS and OSD communication
+ - Current sensor: 165A cont. / 240A peak
+ - FC BEC: 5V 3A cont. (for FC & peripherals)
+ - FPV BEC: 9V/12V 5A (selectable)
+ - Dimensions:  54 x 36 x 13 mm
+
+## Pinout
+
+![TBS Lucid H7 Wing Board Top](Top.png "TBS Lucid H7 Wing Top")
```

Comment:
please rename images to something that includess the autopilot name to avoid wiki conflicts